carapace-server/src/main/java/org/carapaceproxy/server/certiticates/DynamicCertificatesManager.java

```diff
@@ -325,7 +325,10 @@ private void certificatesLifecycle() {
                     groupMembershipHandler.fireEvent(EVENT_CERT_AVAIL_CHANGED);
                 }
             } catch (AcmeException | IOException | ConfigurationStoreException | GeneralSecurityException | IllegalStateException ex) {
+                LOG.log(Level.SEVERE, null, ex);            
```

Comment:
this is okay, we have to leave this line

Reply:
please also add a message like 'Error while handling dynamic certificate xxxx"....